core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -140,7 +140,6 @@
    * @return this
```

Comment:
`* <p>Should only used by tests.` need be revised? The grammar is also incorrect.